README.md

```diff
@@ -1,93 +1,93 @@
 
-# Signalen
+# Signalen frontend
 
-This project is based on the react boilerplate that was adapted for the needs of the municipality of Amsterdam.
-See https://github.com/Amsterdam/react-boilerplate.git for more information on the react boilerplate
+This project provides a web frontend for Signalen, an application that helps cities manage and prioritize nuisance reports.
 
-## Requirements
+## Setup a development environment
 
-  -  npm >= 6.11
-  -  node 12.18
+Make sure you installed the following versions on your machine:
 
-This project relies on the[ `leaflet-headless`](https://www.npmjs.com/package/leaflet-headless) package. On OSX, you'll need to have [`cairo`](https://formulae.brew.sh/formula/cairo), [`pango`](https://formulae.brew.sh/formula/pango) and [`cmake`](https://formulae.brew.sh/formula/cmake) installed.
+-  npm >= 6.11
+-  node 12.18
 
-## Installation
+For testing this project relies on the[ `leaflet-headless`](https://www.npmjs.com/package/leaflet-headless) package. On OSX, you'll need to have [`cairo`](https://formulae.brew.sh/formula/cairo), [`pango`](https://formulae.brew.sh/formula/pango) and [`cmake`](https://formulae.brew.sh/formula/cmake) installed.
```

Comment:
Actually, it doesn't rely on `leaflet-headless` anymore!

Reply:
Will change this